.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: ci
+
+on:
+   workflow_dispatch:
+   schedule:
+     - cron: 0 0 1 * 0     # monthly
+   pull_request:
+     branches:
+       - main
+       - development
+
+jobs:
+  tox:
+    strategy:
+      matrix:
+        os: [ubuntu-latest] #[ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.10'] #['3.8', '3.9', '3.10']
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
```

Comment:
I'm not sure this is actually necessary -- I don't have this in `plenoptic`, and the branch of the PR is checked out each time. It's unclear to me what it does

Reply:
after digging into this it seems that ref: ${{ github.ref }} is of no use with this action; it would behave in the same way if I omit it. I am removing it